magento/magento2
# updated abstractSource.php to fix 24698

getOptionId() should return id/value on basis of label, as of now returning value as well on basis of value only.
This is causing mismatch in some scenario.





### Description (*)


### Fixed Issues (if relevant)

1. magento/magento2#24698: getOptionId() - parameter input is not specific to option value or option label

### Manual testing scenarios (*)

Have any select type attribute of product with options in below mannner
Have an option with label as "09" and suppose its value assigned by magento is "60".
Now later another option has label "60" whose value assigned is any value by magento.
Now Try to get the option id using getOptionId() method in \module-eav\Model\Entity\Attribute\Source\AbstractSource.php
Expected result : The getOptionId should return optionId or value when we pass option label as parameter.
### Questions or comments


### Contribution checklist (*)
